src/WebJobs.Script/Description/CSharp/FunctionAssemblyLoadContext.cs

```diff
@@ -36,13 +39,21 @@ public ImmutableArray<Assembly> LoadedAssemblies
             }
         }
 
+        public TraceWriter TraceWriter
+        {
+            get
+            {
+                return _traceWriter;
+            }
+        }
+
         public Uri FunctionBaseUri
         {
             get
             {
                 if (_functionBaseUri == null)
                 {
-                    _functionBaseUri = new Uri(Path.GetDirectoryName(Metadata.Source), UriKind.RelativeOrAbsolute);
+                    _functionBaseUri = new Uri(Path.GetDirectoryName(Metadata.Source) + "\\", UriKind.RelativeOrAbsolute);
```

Comment:
So the issue was the paths were missing the terminal '\' and were therefore not valid?